### PR TITLE
[IMP] Convert, Relicense, and Upgrade

### PR DESCRIPTION
**Pull request by lasley**
* Convert to standard Docker image instead of Clouder
* Relicense to Apache 2
* Fix cert generation
* Add support for intermediate/root switch off
* Add env vars for config locations
* Separate the trusted certs from the cert store
* Add a docker-compose for full PKI
* Add db generation

### CODE REVIEW

File: bin/init-db [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+def get_config():
+    """ Return the database config object. """
+    config = os.path.join(
+        '/etc/cfssl',
+        os.environ.get('DB_CONFIG'),
+    )
+    with open(config, 'r') as fh:
+        return json.load(fh)
+
+def initialize_db(destroy=None):
```
Comment (tedsalmon) [inline bin/init-db:26]: Why allow destroy to be set if you never call this function anywhere outside of the __main__?

**Review by tedsalmon — APPROVED** [review]
LGTM! 👍 
I did have a minor question, but I think I'm just not thinking of the right use case

File: bin/init-db (hunk 1 above)
Comment (lasley) [reply bin/init-db:26]: Why not?

Comment (tedsalmon) [reply bin/init-db:26]: It just seemed superfluous, but it's not breaking anything so __*shrug*__

Comment (lasley) [reply bin/init-db:26]: Only in the context of this being run as a binary. Basically it's an API that's not used, but didn't hurt to add.

**Pull request merged**